Dockerfile

```diff
@@ -1,13 +1,24 @@
 # Mozilla Kinto server
-FROM python:3.5
+FROM debian:sid
+MAINTAINER Storage Team irc://irc.freenode.net/#kinto
+
 ADD kinto /code/kinto
-ADD README.rst /code/
-ADD CHANGELOG.rst /code/
-ADD CONTRIBUTORS.rst /code/
-ADD setup.py /code/
-ADD MANIFEST.in /code/
-RUN pip install cliquet[postgresql,monitoring]
-RUN pip install -e /code
+ADD CHANGELOG.rst README.rst CONTRIBUTORS.rst setup.py MANIFEST.in /code/
 ENV KINTO_INI /etc/kinto/kinto.ini
-RUN kinto --ini $KINTO_INI --backend=memory init
+
+# Install build dependencies, build the virtualenv and remove build
+# dependencies all at once to build a small image.
+RUN \
+    apt-get update; \
+    apt-get install -y python3 libpq5; \
+    apt-get install -y build-essential git python3-dev python3-setuptools libssl-dev libffi-dev libpq-dev; \
```

Comment:
ah oh ok :)

nanonit: you can put it on the same line as python3 and libpq5 then :)